build.sh

```diff
@@ -144,33 +137,36 @@ if (( ${NUMARGS} == 0 )) || hasArg libcuml || hasArg prims || hasArg bench; then
 fi
 
 # Run all make targets at once
+
 MAKE_TARGETS=
-if (( ${NUMARGS} == 0 )) || hasArg libcuml; then
+if hasArg libcuml; then
     MAKE_TARGETS="${MAKE_TARGETS}cuml++ cuml ml ml_mg"
 fi
-if (( ${NUMARGS} == 0 )) || hasArg prims; then
+if hasArg prims; then
     MAKE_TARGETS="${MAKE_TARGETS} prims"
 fi
-if (( ${NUMARGS} == 0 )) || hasArg bench; then
+if hasArg bench; then
     MAKE_TARGETS="${MAKE_TARGETS} sg_benchmark"
 fi
 
+
+# If there are no targets specified when calling build.sh, it will
```

Comment:
If a user just runs `build.sh cuml`, will they also end up building `libcuml`? If so, that could affect devs iterating on Python-only changes.